apigee_edge.services.yml

```diff
@@ -26,7 +26,11 @@ services:
 
   apigee_edge.exception_subscriber:
     class: Drupal\apigee_edge\EventSubscriber\EdgeExceptionSubscriber
-    arguments: ['@url_generator']
+    arguments:
+      - '@http_kernel'
+      - '@logger.channel.php'
```

Comment:
Why not the `@logger.channel.apigee_edge`? Errors should be logged under our module's namespace in this even subscriber.